src/main.rs

```diff
@@ -573,6 +622,14 @@ impl AsyncComponent for App {
                 *self.exit_status_code.lock() = AppStatusCode::Restart;
                 relm4::main_application().quit();
             }
+            AppInput::ShouWindow => {
+                root.show();
+                root.maximize();
+            }
+            AppInput::Quit => {
+                *self.exit_status_code.lock() = AppStatusCode::Exit;
```

Comment:
It is `AppStatusCode::Exit` by default, no need to set it explicitly

Reply:
done